module/VuFind/src/VuFind/Config/Upgrade.php

```diff
@@ -644,17 +644,6 @@ protected function upgradeConfig()
         // Eliminate obsolete config override settings:
         unset($newConfig['Extra_Config']);
 
-        // Update stats settings:
-        if (isset($newConfig['Statistics']['enabled'])) {
```

Comment:
It might make sense to add a new check to the upgrade script so that if anything in the [Statistics] section has been uncommented, we add a warning to the messages array indicating that the feature is gone.